packages/onboard-react/package.json

```diff
@@ -24,14 +24,15 @@
     "react-i18next": "12.0.0"
   },
   "peerDependencies": {
-    "@sovryn/ui": "1.0.6",
+    "@sovryn/ui": "1.0.16",
     "ethers": "5.7.0",
```

Comment:
also add `@sovryn/onboard-injected` as peer dependency, because you are explicitly requesting wallet icons from the that lib.